MANIFEST.in

```diff
@@ -0,0 +1,3 @@
+include msticpy/data/queries/*.yaml
+include msticpy/msticpyconfig.yaml
```

Comment:
Do we just want to import anything .yaml in the base folder for future extensibility?

Reply:
Maybe - I might also want to add a template msticpyconfig.yaml that gives people a bit of a clue what to do. Perhaps also have some kind of warning about the stuff being loaded with no custom config file (and no authentication parameters)